src/main/java/uk/gov/hmcts/ccd/domain/model/search/Field.java

```diff
@@ -5,6 +5,7 @@
 
 public class Field {
     private String id;
+    private String elementPath;
```

Comment:
Do we need to add elementPath to Field? it's a quite specific property needed only for this particular use case. We already have added it to WorkbasketInputField and SearchInputField can't we use that property?